app/main.py

```diff
@@ -1,5 +1,66 @@
+from __future__ import annotations
+
+
 class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.name = name
+        self.weight = weight
+        if coords:
+            self.coords = coords
+        else:
+            self.coords = [0, 0]
```

Comment:
Do not write self.coords using multiline expression, you could implement the same logic using if else
```suggestion
        self.coords = coords if coords else [0, 0]
```
or using `or` 
```suggestion
        self.coords = coords or [0, 0]
```